DataFormats/L1TrackTrigger/interface/TTBV.h

```diff
@@ -148,31 +177,46 @@ class TTBV {
     return *this;
   }
 
-  // manipulation: all bits
+  // manipulation: all bits set to 0
   TTBV& reset() {
     bs_.reset();
     return *this;
   }
+
+  // manipulation: all bits set to 1
   TTBV& set() {
     for (int n = 0; n < size_; n++)
       bs_.set(n);
     return *this;
   }
+
+  // manipulation: all bits flip 1 to 0 and vice versa
   TTBV& flip() {
     for (int n = 0; n < size_; n++)
       bs_.flip(n);
     return *this;
   }
 
-  // manipulation: single bit
+  // manipulation: single bit set to 0
   TTBV& reset(int pos) {
     bs_.reset(pos);
     return *this;
   }
+
+  // manipulation: single bit set to 1
   TTBV& set(int pos) {
     bs_.set(pos);
     return *this;
   }
+
+  // manipulation: multiple bit set to 1
+  TTBV& set(std::vector<int> vpos) {
+    for (int pos : vpos)
+      bs_.set(pos);
+    return *this;
+  }
+
+  // manipulation: single bit flip 1 to 0 and vice versa
```

Comment:
Does comment for TTBV::abs() need to say "biased"? It's not, is it? And comment should make clear abs() also switches to binary format.

Reply:
done

Reply:
The comment for TTBV::abs() at https://github.com/cms-L1TK/cmssw/blob/tschuh/DataFormats/L1TrackTrigger/interface/TTBV.h#L225 seems unchanged?

Reply:
correct. It is the absolute value of biased integers, if you feed unbiased integers you don't get the absolute value. Shall I change "biased absolute value" into "absolute value for biased integers"?